addons/website/models/website.py

```diff
@@ -139,6 +139,9 @@ def _default_favicon(self):
     favicon = fields.Binary(string="Website Favicon", help="This field holds the image used to display a favicon on the website.", default=_default_favicon)
     theme_id = fields.Many2one('ir.module.module', help='Installed theme')
 
+
+    website_design_id = fields.Many2one('website.design', string='Website Design')
```

Comment:
Do not prefix a website field with `website_`. It's a convention that could make sense but none of our field does that here :grimacing: